extensions/rich_text_components/Skillreview/directives/oppia-noninteractive-skillreview.component.ts

```diff
@@ -83,9 +86,11 @@ export class NoninteractiveSkillreview implements OnInit, OnChanges {
   // instance, then open the concept card modal. To determine if the
   // RTE is inside a CKEditor instance, check if the offsetParent
   // element contains the data attribute ckeWidgetId.
-  private _shouldOpenRTEModal(event): boolean {
+  private _shouldOpenRTEModal(event: Event): boolean {
+    const target = <HTMLElement> event.currentTarget;
+    const offsetParent = <HTMLElement> target.offsetParent;
     return (
-      event.currentTarget.offsetParent.dataset.ckeWidgetId ||
+      offsetParent.dataset.ckeWidgetId ? true : false ||
```

Comment:
Why this? And can we do this?
```suggestion
      Boolean(offsetParent.dataset.ckeWidgetId) ||
```

Reply:
The change was made because the values of `offsetParent.dataset.ckeWidgetId` could either be `null` or `string` and not `boolean`. The suggested change is made